src/linagora.esn.videoconference/app/application-menu/application-menu.directive.js

```diff
@@ -0,0 +1,25 @@
+(function(angular) {
```

Comment:
This package should not add a videoconference menu entry in the application grid.